acme/acme/challenges_test.py

```diff
@@ -359,9 +359,12 @@ def test_simple_verify(self, mock_verify_cert):
             cert=mock.sentinel.cert))
         mock_verify_cert.assert_called_once_with(self.msg, mock.sentinel.cert)
 
-    def test_simple_verify_false_on_probe_error(self):
+    @mock.patch('acme.challenges.socket.gethostbyname')
+    def test_simple_verify_false_on_probe_error(self, mock_gethostbyname):
         chall = mock.Mock()
         chall.probe_cert.side_effect = errors.Error
+
+        mock_gethostbyname.return_value = '127.0.0.2'
```

Comment:
Trying to run the unit tests while offline, I got the following error:
```
======================================================================
ERROR: test_simple_verify_false_on_probe_error (acme.challenges_test.DVSNIResponseTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pde/eff/chocolate/chocolate/letsencrypt/acme/acme/challenges_test.py", line 366, in test_simple_verify_false_on_probe_error
    self.chall, self.domain, self.key.public_key()))
  File "/home/pde/eff/chocolate/chocolate/letsencrypt/acme/acme/challenges.py", line 443, in simple_verify
    cert = self.probe_cert(domain=domain, **kwargs)
  File "/home/pde/eff/chocolate/chocolate/letsencrypt/acme/acme/challenges.py", line 383, in probe_cert
    host = socket.gethostbyname(domain)
gaierror: [Errno -3] Temporary failure in name resolution
-------------------- >> begin captured logging << --------------------
```
The above change fixed that issue.